.github/workflows/security-audit.yml

```diff
@@ -0,0 +1,28 @@
+name: Security Audit
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "1 1 */1 * *" # Run once a day
```

Comment:
What's the advantage of `*/1` vs `*` for the day?

GitHub suggests to not start jobs at the hour. In this case it's 1 minute after, where the load is probably still high.

Reply:
i would suggest to run these checks early in the morning european time 04:00, or so.